binary-search-task/script.js

```diff
@@ -0,0 +1,36 @@
+function binarySearch(arr, number) {
+  if (arr.length == 0) {
```

Comment:
imo this and next `if`s are not required,  you could configure your calls to avoid these cases